tests/core/test_expectation_suite.py

```diff
@@ -7,24 +7,155 @@
 from ruamel.yaml import YAML
 
 from great_expectations import DataContext
+from great_expectations import __version__ as ge_version
 from great_expectations.core.expectation_configuration import ExpectationConfiguration
 from great_expectations.core.expectation_suite import (
     ExpectationSuite,
     expectationSuiteSchema,
 )
 from great_expectations.core.usage_statistics.events import UsageStatsEvents
+from great_expectations.exceptions import InvalidExpectationConfigurationError
 from great_expectations.util import filter_properties_dict
 
 
 @pytest.fixture
-def exp1():
+def test_expectation_suite_name():
```

Comment:
I'd prefer not to name non-tests with the `test_` prefix since I'm use to thinking everything with the `test_` prefix is a test. I don't know how `pytest` handles this (ie will it run it as a test in addition to using it as a fixture).